UnityProject/Assets/Scripts/Items/Medical/RollerBedSpriteChange.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Objects;
+
+public class RollerBedSpriteChange : MonoBehaviour
+{
+    ///<summary>
```

Comment:
Wrong placement of summery, put it ontop of the class not the `BuckleInteract` variable